src/test/java/seedu/duke/commands/FindCommandTest.java

```diff
@@ -7,16 +7,24 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+//@@author DavidVin357
 public class FindCommandTest {
-
+    private String getExpectedFeedback(int expectedNumber) {
+        String FOUND_FOOD = "Found " + expectedNumber + " of food items";
+        String NOT_FOUND = "No food found for such query";
+        if (expectedNumber > 0) {
+            return FOUND_FOOD;
+        }
+        return NOT_FOUND;
+    }
```

Comment:
Naming of constants with UPPERCASE is good, but maybe should add `final` when initializing it.